src/hooks/useOnAvatarPress.ts

```diff
@@ -85,10 +82,12 @@ export default ({ screenType = 'transaction' }: UseOnAvatarPressProps = {}) => {
   }, [accountColor, accountName, navigate, screenType]);
 
   const onAvatarChooseImage = useCallback(async () => {
-    const image = await openPicker({
-      cropperCircleOverlay: true,
-      cropping: true,
+    const result = await openPicker({
+      allowsEditing: true,
```

Comment:
expo-image-picker doesn't support circle overlay. I implemented it for android [here](https://github.com/expo/expo/pull/38148), but for iOS it uses the system picker and it is not supported.